### PR TITLE
Add a blogpost about 1.11.2 bugs

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto roidelapluie@inuits.eu

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
Looks good to me - helping people work around bugs is useful. Anyone else have any concerns over publishing?

File: _posts/2016-06-20-foreman-1.11-dhcp-bugs.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Foreman 1.11.2 DGCP bugs
```
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-1.11-dhcp-bugs.md]: Typo? DHCP right?

Could you make this clearer about the focus of the content? Perhaps "Working around the Foreman 1.11.2 DHCP bugs"?

File: _posts/2016-06-20-foreman-1.11-dhcp-bugs.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+date: 2016-06-20
+author: Julien Pivotto
+tags:
+- foreman
+excerpt: |
+  The latest 1.11.2 maintenance release introduced a unfortunate combination of
+  regressions regarding DHCP.
```
Comment (GregSutcliffe) [inline _posts/2016-06-20-foreman-1.11-dhcp-bugs.md]: Maybe add "Here's how to work around them" as an extra sentence?

**Comment by GregSutcliffe** [comment]
@roidelapluie 2 small comments, then I can merge ;)

**Comment by roidelapluie** [comment]
done

**Comment by GregSutcliffe** [comment]
Merged, thanks @roidelapluie!

**Pull request closed without merging**